fillers/eips/eip4844/excess_data_gas.py

```diff
@@ -696,7 +696,9 @@ def generate(self) -> BlockchainTest:
                 blob_versioned_hashes=[
                     to_hash_bytes(x) for x in range(self.blobs_per_tx)
                 ],
-                error=self.tx_error if tx_i == (self.tx_count - 1) else None,
+                error=self.tx_error
+                if tx_i == (self.tx_count - 1) and self.blobs_per_tx != 0
```

Comment:
I don't understand this check, are we not getting an error if `self.blobs_per_tx == 0` ?